### ISSUE TITLE
Low Temperature Night Mode does not stay enabled

### ISSUE THREAD

**Issue by eyv** [anthonya1999/GoodNight#27]
Greetings,

I have Goodnight set to change color to 2700 at 5 PM and change back at 7 AM. That works great. Now I'm trying to enable low temperature night mode from 9:30 PM yo 6:30 AM. When I flip the switch to enabled, everything is OK, but when I hit the back arrow to leave the temperature settings screen, and then go back to the temperature settings, low temperature night mode is disabled again. There doesn't seem to be anything I can do to get it to stay in the enabled position.

iPhone 6 plus, iOS 9.1, background refresh on, notifications off, locations while using.

Thanks.

**Comment by mariokorte** [comment]
Did you use the latest version? There was an important commit 04274b2351a8c7ad6c2451a2d3dce8327abdb8bf yesterday which should fix all the bug with night mode.

**Comment by anthonya1999** [comment]
I am going to close this for the time being unless the original person who posted the issue still has it.

**Comment by dhmartin82** [comment]
I'm actually having this same issue. I'm on the latest version (as of Dec 4). Night Mode will not stay enabled for me either. Symptoms are the same as described in original post.

I got the notification at the specified time that "It's time to enable Night Mode", but this does not trigger a temperature change. I went into the app and manually enabled Night mode, which then triggered the change to 2000K (the settings toggle for Night Mode then went back to disabled when I left that section).

Thanks!
